src/tests/e2e/receiveTransfer.test.e2e.js

```diff
@@ -120,14 +125,14 @@ describe('Receive transfer tests', () => {
 
     const transferId = pendingReceive[`${walletType}_${cryptoType}`]
     while (true) {
-      await page.waitFor(15000) // 15 seconds
       const transferData = await getTransfer({ transferId: transferId })
       sendTxState = getTransferState(transferData)
       if (sendTxState === 'SEND_CONFIRMED_RECEIVE_NOT_INITIATED') {
         var { receivingId } = transferData
         break
       }
       log.info('Waiting for send tx to be confirmed, current state: ', sendTxState)
+      await page.waitFor(15000) // 15 seconds
```

Comment:
This can speed up the test in some cases. e.g. we send out three txs consecutively, there's a possibility that we don't need to wait 15 seconds for the second and third confirmation.